gmime/test_data/inline-attachment_multipart.eml

```diff
@@ -2,88 +2,91 @@ MIME-Version: 1.0
 Received: by 10.220.115.205 with HTTP; Tue, 27 May 2014 13:46:04 -0700 (PDT)
```

Comment:
not sure why github thinks i renamed the file, but _muteted file was deleted and this new file was created. they are both for different purpose